File: examples/fizzbuzz/fizzbuzz.cpp
#include <iostream>

int main()
{
    int i = 1;
    
    while (i <= 100)
    {
        std::string output = "";
        
        if (i % 3 == 0)
        {
            output += "Fizz";
        }
        if (i % 5 == 0)
        {
            output += "Buzz";
        }
        
        if (output == "")
        {
            std::cout << i << std::endl;
        }
        else
        {
            std::cout << output << std::endl;
        }
        
        i += 1;
    }
}

File: examples/fizzbuzz/fizzbuzz.py
i = 1

while i <= 100:
    output = ""
    
    if i % 3 == 0:
        output += "Fizz"
    if i % 5 == 0:
        output += "Buzz"
    
    if output == "":
        print(i)
    else:
        print(output)
    
    i += 1
